regression-tests/daaltkregtests/testcases/scoretests/pca_test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from daaltkregtests.lib import daaltk_test
+from daaltkregtests.lib import score_utils
+
+
+class PrincipalComponent(daaltk_test.DaalTKTestCase):
+
+    def setUp(self):
+        super(PrincipalComponent, self).setUp()
+        schema = [("X1", int),
+                  ("X2", int),
+                  ("X3", int),
+                  ("X4", int),
+                  ("X5", int),
+                  ("X6", int),
+                  ("X7", int),
+                  ("X8", int),
+                  ("X9", int),
+                  ("X10", int)]
+        pca_traindata = self.get_file("pcadata.csv")
```

Comment:
Yes, it is the same dataset as that used in sparktk for testing. I'll add the dataset to this repo